api/githubsearch.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
Add comments to every service, rpc, and message. Also, great idea to add validation rules to each field.